diff --git a/index.cjs b/index.cjs
index 008dcec48b8531fcd3d1bef251fc089b3d07ec0d..120a885865a443b5b5f4cc6c5d63c2fd99b2e75b 100644
--- a/index.cjs
+++ b/index.cjs
@@ -175,3 +175,4 @@ var src_default = react;
 // <stdin>
 module.exports = src_default;
 module.exports.default = src_default;
+module.exports.react = src_default;
diff --git a/index.d.ts b/index.d.ts
index 725bd1c438d622c3177eb7daae118a825d2372e5..5ee3f536763b4c8813014c4ba392b5fc0f392433 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -18,3 +18,4 @@ type Options = {
 };
 declare const react: (_options?: Options) => PluginOption[];
 export default react;
+export { react };
diff --git a/index.mjs b/index.mjs
index 4bfa11412c9033247527d50ab406b34a86302f50..77d2c2b61f26ba339121d868f3d33604dc3adcb5 100644
--- a/index.mjs
+++ b/index.mjs
@@ -173,5 +173,6 @@ var silenceUseClientWarning = (userConfig) => ({
 });
 var src_default = react;
 export {
-  src_default as default
+  src_default as default,
+  src_default as react
 };